packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "2.138.0",
+  "version": "2.138.0-fb-subfolder-media.0",
```

Comment:
FYI, I think Karl is in the process of merging a patch fix and will use `2.138.1` so yours will likely be `2.138.2`